### PR TITLE
Guidelines for raising FlowForge in NR community

### PR DESCRIPTION
**Pull request by Steve-Mcl**
Add guidelines for raising FlowForge in the Node-RED community

### CODE REVIEW

File: company/communication.md [hunk 1]
```diff
@@ -39,3 +39,13 @@ scroll to a notes section.
 
 Coffee calls are social of nature and thus are the exception to the rule that
 each meeting should have an agenda.
+
+
+## Node-RED community interations
```
Comment (ZJvandeWeg) [inline company/communication.md:44]:
```suggestion
## Node-RED community interactions
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]
One typo fix, but looks good to me! Thanks @Steve-Mcl!

File: company/communication.md (hunk 1 above)
Comment (Steve-Mcl) [reply company/communication.md]: done

**Review by ZJvandeWeg — APPROVED** [review]

**Comment by knolleary** [comment]
👍🏻

**Pull request merged**